compiler/rustc_driver_impl/Cargo.toml

```diff
@@ -19,6 +19,7 @@ rustc_errors = { path = "../rustc_errors" }
 rustc_expand = { path = "../rustc_expand" }
 rustc_feature = { path = "../rustc_feature" }
 rustc_fluent_macro = { path = "../rustc_fluent_macro" }
+rustc_hir = { path = "../rustc_hir" }
```

Comment:
Is this used? can't find any new `rustc_hir` around.

Reply:
Doesn't look like it, removed.